### ISSUE TITLE
G transform="translate()" not working on Android

### ISSUE THREAD

**Issue by draperunner** [react-native-community/react-native-svg#1091]
# Bug

I am displaying the following SVG in my app. It looks good on iOS, but not right at all on Android. I'll update the issue with a screenshot of the erroneous icon when I can.

This did look good on Android when I was using v9.5.3 of this library, but not on the current version. 9.8.4.

Maybe `<G transform="translate(0,-942.36218)">` is broken in the last versions?

The icon: https://thenounproject.com/search/?q=mountain&creator=2685385&i=1185303

Erroneous views:

![svg-error1](https://user-images.githubusercontent.com/4339443/64474966-74f6ec00-d17c-11e9-9f1e-4ea8c154e2e7.png)

![svg-error](https://user-images.githubusercontent.com/4339443/64474969-77594600-d17c-11e9-98ef-8fecb9e1c837.png)

## Environment info

React native info output:

```bash
System:
    OS: macOS 10.14.6
    CPU: (8) x64 Intel(R) Core(TM) i7-6820HQ CPU @ 2.70GHz
    Memory: 2.49 GB / 16.00 GB
    Shell: 3.2.57 - /bin/bash
  Binaries:
    Node: 10.13.0 - ~/.nvm/versions/node/v10.13.0/bin/node
    Yarn: 1.12.3 - ~/.yarn/bin/yarn
    npm: 6.9.0 - ~/.nvm/versions/node/v10.13.0/bin/npm
    Watchman: 4.9.0 - /usr/local/bin/watchman
  SDKs:
    iOS SDK:
      Platforms: iOS 12.4, macOS 10.14, tvOS 12.4, watchOS 5.3
    Android SDK:
      API Levels: 23, 25, 26, 27, 28
      Build Tools: 23.0.1, 25.0.2, 25.0.3, 26.0.1, 26.0.2, 26.0.3, 27.0.0, 27.0.3, 28.0.2, 28.0.3
      System Images: android-23 | Intel x86 Atom_64, android-23 | Google APIs Intel x86 Atom_64, android-25 | Google APIs Intel x86 Atom_64, android-25 | Google Play Intel x86 Atom, android-26 | Google APIs Intel x86 Atom_64, android-27 | Google APIs Intel x86 Atom, android-28 | Google APIs Intel x86 Atom, android-28 | Google Play Intel x86 Atom
      Android NDK: 17.1.4828580
  IDEs:
    Android Studio: 3.2 AI-181.5540.7.32.5056338
    Xcode: 10.3/10G8 - /usr/bin/xcodebuild
  npmPackages:
    react: 16.8.6 => 16.8.6
    react-native: 0.60.5 => 0.60.5
  npmGlobalPackages:
    eslint-plugin-react-native-a11y: 1.2.0
    eslint-plugin-react-native: 3.6.0
    react-native-cli: 2.0.1
    react-native-git-upgrade: 0.2.7
```

Library version: 9.8.4

## Steps To Reproduce

1. Copy the SVG code above into a React Native app
2. Run it on Android
...

Describe what you expected to happen:

I expected the icon to be displayed correctly on Android.

## Reproducible sample code

```
import React from 'react'
import Svg, { Path, G } from 'react-native-svg'

function MountainsIcon({ size = 16, color = '#000000', style }) {
  return (
    <Svg
      fill={color}
      height={size}
      viewBox="0 0 100 125"
      width={size}
      style={style}
    >
      <G transform="translate(0,-942.36218)">
        <Path
          d="m 57.119073,970.31534 c -0.644,0.008 -1.3091,0.39348 -1.7812,1.09375 l -17.9375,30.78121 -5.3125,-9.15621 c -0.3361,-0.5832 -0.9837,-0.97415 -1.6562,-1 -0.7248,-0.0288 -1.4504,0.37157 -1.8126,1 L 6.4940731,1031.0966 c -1.1914,1.9778 0.091,3.3158 1.9063,3.3125 l 83.6250019,0 c 1.3917,-0.01 2.4145,-1.7946 1.7187,-3 L 58.806673,971.40909 c -0.4205,-0.73137 -1.0435,-1.10153 -1.6876,-1.09375 z m -0.9374,12.8125 a 2.0001998,2.0001998 0 0 1 1.8437,1.03125 l 16.375,27.96871 a 2.0098544,2.0098544 0 1 1 -3.4687,2.0313 l -16.375,-28.00001 a 2.0001998,2.0001998 0 0 1 1.625,-3.03125 z"
          fill={color}
          fill-rule="evenodd"
          overflow="visible"
        />
      </G>
    </Svg>
  )
}

export default MountainsIcon
```

**Comment by draperunner** [comment]
I have now worked my SVG through Inkscape and SVGOMG in order to get rid of the `<G transform="translate(0,-942.36218)">`, and it now works. So I can confirm that the translation is the problem here.

This works:
```
import React from 'react'
import Svg, { Path } from 'react-native-svg'

function MountainsIcon({ size = 16, color = '#000000', style }) {
  return (
    <Svg
      fill={color}
      height={size}
      viewBox="0 0 100 125"
      width={size}
      style={style}
    >
      <Path
        fill-rule="evenodd"
        d="M57.12 30.906c-.645.008-1.31.394-1.782 1.094L37.4 62.78l-5.312-9.156a2.039 2.039 0 0 0-1.656-1 2.044 2.044 0 0 0-1.813 1L6.494 91.687C5.303 93.665 6.585 95.003 8.4 95h83.625c1.392-.01 2.415-1.795 1.72-3L58.806 32c-.42-.731-1.044-1.102-1.688-1.094zm-.938 12.813a2 2 0 0 1 1.843 1.03L74.4 72.72a2.01 2.01 0 1 1-3.468 2.031l-16.375-28a2 2 0 0 1 1.625-3.031z"
        color={color}
        overflow="visible"
      />
    </Svg>
  )
}

export default MountainsIcon
```

**Comment by msand** [comment]
@draperunner can you try https://github.com/react-native-community/react-native-svg/commit/88b0c584a4a1768c0d08b5ba8c82375503286e37

**Comment by msand** [comment]
published v9.8.5

**Comment by draperunner** [comment]
I now tried the problematic icon with v9.8.5, and it looks good! So v9.8.5 fixed it. But I think I'll stick with my non-`transform` icon though. ;)

Thanks!
